bin/scripts/script.js

```diff
@@ -1,27 +1,34 @@
-const { ipcRenderer } = require('electron');
+const { ipcRenderer } = require("electron");
 
-document.addEventListener("DOMContentLoaded", function() {
-var myInput = document.getElementById("input");
-var naratorOutput = document.getElementById("output");
-var characterpicture = document.getElementById("character_picture_img");
-myInput.addEventListener("keydown", function(event) {
-  if (event.key === "Enter") {
-    const inputValue = myInput.value;
-    handleInput(inputValue);
-  }
-});
+document.addEventListener("DOMContentLoaded", function () {
+  var myInput = document.getElementById("input");
+  var narratorGreeting = document.getElementById("greeting");
+  var characterPicture = document.getElementById("character_img");
+  myInput.addEventListener("keydown", function (event) {
+    if (event.key === "Enter") {
+      const inputValue = myInput.value;
+      handleInput(inputValue);
+    }
+  });
 
-function handleInput(input) {
+  function handleInput(input) {
+    // do something with the input
+    myInput.value = "";
+    let num = input.charAt(9);
+    if (input == "yes" /*missing scripts for story and comparisons*/) {
+      narratorGreeting.innerHTML = "missing scripts for story and comparisons";
+    }
+    if (
+      input ==
+      String("character" + num) /*missing scripts for story and comparisons*/
+    ) {
+      characterPicture.src = "character/" + input + "/image/character.png";
+    }
 
-  // do something with the input
-  myInput.value = ""
-  let num = input.charAt(9) 
-  if(input=="yes" /*missing scripts for story and comparisons*/){naratorOutput.innerHTML = "missing scripts for story and comparisons"}
-  if(input==String("character"+num) /*missing scripts for story and comparisons*/){characterpicture.src = "character/"+input+"/image/character.png"}
-  
-ipcRenderer.send("log", "Input received: " + input,"INFO","script.js:22");
+    ipcRenderer.send("log", "Input received: " + input, "INFO", "script.js:22");
```

Comment:
Can you explain this line? So I understand

Reply:
ipcRender sends to the main js file that it should log data provided to it, script.js:22 means info is on line 22, but by changes you moved it to line 28, its also just a placeholder as just so that I dont forget how I made logging errors/infos to log files done